js/screens/ChildProfile/ChildProfile.js

```diff
@@ -26,8 +26,11 @@ const ChildProfile = ({
           action={
             <OutlinedButton
               title="more info"
-              pressed={() => {}}
+              pressed={() => {
+                navigation.navigate('ChildInfo', { child: child });
+              }}
               color={colors.teal}
+              titleStyle={{ marginTop: -5, marginBottom: -7 }}
```

Comment:
Please do not use negative margins. Find an alternative approach.

Reply:
the only approach I could come up with is to not use the designer's font. the font comes with its own built-in padding. the margin removes it to make the button match the designer's vision.